code/controllers/subsystem/points.dm

```diff
@@ -68,16 +69,14 @@ SUBSYSTEM_DEF(points)
 				containsname[path]["count"]++
 		supply_packs_contents[pack] = list("name" = P.name, "container_name" = initial(P.containertype.name), "cost" = P.cost, "hidden" = P.hidden, "contains" = containsname)
 
-	for(var/typepath in subtypesof(/datum/supply_export))
-		var/datum/supply_export/E = new typepath()
-		GLOB.exports_types[E.export_obj] = E
```

Comment:
This removes supply export, that were used in export history.

Making them faction req compatible would be too much work, for something that is actually completly useless

Reply:
there should still be some tracker for the exports